### ISSUE TITLE
ObjectManager: implement InterfacesAdded and InterfacesRemoved

### ISSUE THREAD

**Issue by acrisci** [altdesktop/python-dbus-next#36]
https://dbus.freedesktop.org/doc/dbus-specification.html#standard-interfaces-objectmanager

```
org.freedesktop.DBus.ObjectManager.InterfacesAdded (OBJPATH object_path,
    DICT<STRING,DICT<STRING,VARIANT>> interfaces_and_properties);
org.freedesktop.DBus.ObjectManager.InterfacesRemoved (OBJPATH object_path,
    ARRAY<STRING> interfaces);
```

> The InterfacesAdded signal is emitted when either a new object is added or when an existing object gains one or more interfaces. The InterfacesRemoved signal is emitted whenever an object is removed or it loses one or more interfaces. The second parameter of the InterfacesAdded signal contains a dict with the interfaces and properties (if any) that have been added to the given object path. Similarly, the second parameter of the InterfacesRemoved signal contains an array of the interfaces that were removed. Note that changes on properties on existing interfaces are not reported using this interface - an application should also monitor the existing PropertiesChanged signal on each object.

**Comment by michallowasrzechonek-silvair** [comment]
@rafalgajda-silvair

**Comment by acrisci** [comment]
fixed in #37
